dist/index.bundle.js

```diff
@@ -533,20 +544,17 @@ class Task {
     localStorage.setItem('tasks', JSON.stringify(this.list));
   }
 
-  RemoveTask(index) {
-    this.list = this.list.filter((task) => task.index !== index);
-    this.list = this.list.map((t) => {
-      if (t.index > index) {
-        t.index -= 1;
-      }
-      return t;
+  RemoveTask() {
+    this.list = this.list.filter((todo) => todo.id === null);
+    this.list.forEach((todo, index) => {
+      todo.index = index + 1;
     });
     localStorage.setItem('tasks', JSON.stringify(this.list));
   }
```

Comment:
- [ ] Great job so far :clap: however there is a problem when you delete an item from the list, I can see that every time you delete an item it also deletes all other items from your storage and empties the local storage "tasks" array instead of deleting only the concerned